lua/svartafanan/main.lua

```diff
@@ -16,7 +17,24 @@ function M.main(cube_size)
 	local scramble_string = table.concat(scramble, " ")
 
 	-- Print the scramble to the buffer
-	window.update(win, buf, scramble_string)
+	window.update(scramble_string, 3)
+
+	window.update("Press <space> to start/stop the timer", 5)
+	window.update("Press q to close the window", 6)
+
+	window.update_timer(0)
+
+	vim.keymap.set("n", "<space>", function()
+		if timer.active then
+			timer.stop_timer()
+		else
+			timer.start_timer()
+		end
+	end, { buffer = window.buf, desc = "svartafanan: start timer", nowait = true })
+
+	vim.keymap.set("n", "q", function()
```

Comment:
I've been thinking of adding this to all my plugins as I've realized that I try to press 'q' to quit a lot of times.